### PR TITLE
Removes caps from many improper items

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request

Removes caps from many improper items.
For example - Heated Rebar Crossbow -> heated rebar crossbow

I've kept a lot of items untouched because for some reason or another it felt nicer to me that they be, uh, capsed. For example Hat of the Honkmother, anything that has 'weight' to it.

There's also some minor name changes that I think fit better, such as Rebar Storage Quiver -> rebar quiver. I mean, we already know it's for storage. That's the whole point of quivers.
## Why It's Good For The Game

This is one of my biggest pet peeves, Randomly Capitalized Items For No Reason. It looks sloppy, awkward, and half-assed. This PR brings a lot of the worst offenders in line. if your name isn't a proper noun it almost never should have caps. 
## Changelog
:cl:
spellcheck: Removes caps from many improper items
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 1]
```diff
@@ -192,7 +192,7 @@
 		spread = 50
 
 /obj/item/gun/ballistic/rifle/rebarxbow
-	name = "Heated Rebar Crossbow"
+	name = "heated rebar Crossbow"
```
Comment (lessthnthree) [inline code/modules/projectiles/guns/ballistic/rifle.dm:195]:
```suggestion
	name = "heated rebar crossbow"
```

File: code/modules/events/holiday/easter.dm [hunk 2]
```diff
@@ -66,7 +66,7 @@
 
 //Bunny Suit
 /obj/item/clothing/head/costume/bunnyhead
-	name = "Easter Bunny Head"
+	name = "Easter Bunny head"
```
Comment (MrMelbert) [inline code/modules/events/holiday/easter.dm:69]: Should have `\improper`?

File: code/modules/events/holiday/easter.dm [hunk 3]
```diff
@@ -75,7 +75,7 @@
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE|HIDEHAIR|HIDEFACIALHAIR|HIDESNOUT
 
 /obj/item/clothing/suit/costume/bunnysuit
-	name = "Easter Bunny Suit"
+	name = "Easter Bunny suit"
```
Comment (MrMelbert) [inline code/modules/events/holiday/easter.dm:78]: Same

File: code/modules/events/holiday/easter.dm [hunk 4]
```diff
@@ -88,7 +88,7 @@
 
 //Bunny bag!
 /obj/item/storage/backpack/satchel/bunnysatchel
-	name = "Easter Bunny Satchel"
+	name = "Easter Bunny satchel"
```
Comment (MrMelbert) [inline code/modules/events/holiday/easter.dm:91]: Same

Comment (carlarctg) [reply code/modules/events/holiday/easter.dm:91]: those are on purposeish but ok
```suggestion
	name = "easter bunny satchel"
```

File: code/modules/events/holiday/easter.dm (hunk 3 above)
Comment (carlarctg) [reply code/modules/events/holiday/easter.dm:78]:
```suggestion
	name = "easterbunny suit"
```

File: code/modules/events/holiday/easter.dm (hunk 2 above)
Comment (necromanceranne) [reply code/modules/events/holiday/easter.dm:69]: If this is capitalized, why are the other parts of the costume not capitalized?

**Pull request merged**